cluster/cluster.go

```diff
@@ -388,10 +388,10 @@ func (cluster *Cluster) CheckClusterError(remoteOutput *RemoteOutput, finalErrMs
 		case func(content int) string:
 			content := failedCommand.Content
 			host := cluster.GetHostForContent(content)
-			gplog.Verbose("%s on segment %d on host %s %s", getMessage(content), content, host, errStr)
+			gplog.Custom(gplog.LOGERROR, gplog.LOGVERBOSE, "%s on segment %d on host %s %s", getMessage(content), content, host, errStr)
```

Comment:
I find this very unusual (at least from the perspective of Java's Log4J logging standard) that a single message gets logged and displayed with different logging levels (called `verbosity` here). IMO, the purpose of the log is to capture what is shown to the user (and maybe more context, if needed), so that during troubleshooting it would be possible to see what the user saw and more context around it to assist in troubleshooting. If this is not presented as an "ERROR" to the shell, why would it present as an "ERROR" to the log file ?

I understand the desire to limit the amount of info in the log file, but that is usually achieved by setting different logging level thresholds (called `verbosity` here) for file appender vs console appender (called `shell` here).

Having said that, PXF is using neither this particular function nor the Custom() function, so this does not seem to have an impact on the PXF project.